app/models/book_author.rb

```diff
@@ -0,0 +1,4 @@
+class BookAuthor < ActiveRecord::Base
```

Comment:
Naming is hard...credit actually made me think of it in a financial sense before I thought about it further....but I'm fine with it if the group decides that's a name for it.  Maybe we should talk about it on the call tonight.